From 5a2eec678cad5737b1e29a54f5ca3650e2e592bd Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Wed, 9 May 2018 19:37:29 +0800
Subject: [PATCH 4/4] usb: common: Fix the misleading indentation in
 of_usb_get_maximum_speed()

Fix the misleading indentation in of_usb_get_maximum_speed() introduced
by commit de3841dfd839 ("usb: dwc3: add function of_usb_get_dr_mode").

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/usb/common/common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/common/common.c b/drivers/usb/common/common.c
index 176dee0..f4165f2 100644
--- a/drivers/usb/common/common.c
+++ b/drivers/usb/common/common.c
@@ -151,7 +151,7 @@ enum usb_device_speed of_usb_get_maximum_speed(struct device_node *np)
 		if (strcmp(maximum_speed, speed_names[i]) == 0)
 			return i;
 
-		return USB_SPEED_UNKNOWN;
+	return USB_SPEED_UNKNOWN;
 }
 EXPORT_SYMBOL_GPL(of_usb_get_maximum_speed);
 
-- 
1.8.5.2.233.g932f7e4

